__tests__/requests.test.js

```diff
@@ -31,3 +31,17 @@ describe('404 error handling', () => {
       });
   });
 });
+
+describe("get API endpoints", () => {
+  it('GET - status: 200 - responds with all api endpoints', () => {
+      return request(app)
+      .get("/api")
+      .expect(200)
+      .then((res) => {
+          expect(typeof res.body).toBe('object')
+          expect(typeof res.body["GET /api"].description).toBe('string')
+          expect(res.body.hasOwnProperty('GET /api/categories'))
```

Comment:
Rather than having to go back and add to these, The test can simply require in the same JSON file and assert it is sent as a response :) this will make the code tidier but also will make your life a bit easier, than having to go back and update this test